index.html

```diff
@@ -1,32 +1,22 @@
 <!doctype html>
 <html>
-<head>
+  <head>
     <title>We'll be right back!</title>
-    <style type="text/css">
-        body {
-            font-family: arial, sans-serif;
-            font-size: 16pt;
-        }
-        #content {
-            width: 960px;
-            margin: 5em auto;
-        }
-        #logo {
-            margin-top: 10em;
-        }
-    </style>
-</head>
-<body>
-<div id="content">
-    <h1>We&rsquo;ll be right back!</h1>
-
-
-    <p>This web site is unavailable at the moment because we&rsquo;re doing
-    some essential maintenance. Sorry for the inconvenience!</p>
-
-    <p>Please check back soon.</p>
-
-    <p id="logo"><a href="https://www.zooniverse.org/"><img src="logo.png" alt="Zooniverse"></a></p>
-</div>
-</body>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://fonts.googleapis.com/css?family=Karla" rel="stylesheet">
+  </head>
+  <body>
+    <img src="img/shape.svg">
+    <div class="container">
+      <a href="https://www.zooniverse.org/">
+        <img src="img/zooniverse-logo.svg" class="zooniverse-logo">
+      </a>
+      <div class="title">OLD WEATHER WHALING</div>
+      <h1 class="maintenance-msg">Down for maintenance</h1>
+      <p class="old-weather-whaling-text">
+        Old Weather Whaling is currently down for maintenance. <br>
```

Comment:
Something wrong with the period at the end of the sentence. Delete it and make sure it's just text?